### PR TITLE
Add separate doc for different version of webpack

### PR DESCRIPTION
**Pull request by diondirza**
This change is [Reviewable](https://reviewable.io/reviews/shakacode/font-awesome-loader/10)

### CODE REVIEW

**Comment by justin808** [comment]
Thanks @diondirza! @alexfedoseev and I will take a look!

File: docs/usage-webpack1.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+Font awesome font urls are of the format `[dot][extension]?=[version-number]`, for example `.woff?v=4.2.0`
+
+The Regex for font types are adjusted to support these formats. Regex also support urls ending with .woff, .ttf, .eot and .svg (Used by Bootstrap).
+
+## Complete Font-Awesome
+
+To use the complete font-awesome package including all styles with the default settings:
+
+``` javascript
+require("font-awesome-loader");
```
Comment (alexfedoseev) [inline docs/usage-webpack1.md]: Or modern way:

```js
import 'font-awesome-loader';
```

File: docs/usage-webpack2.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
+# Setup for Webpack 2
+
+To properly load font-awesome fonts, you need to install font-awesome as dependencies
+
+``` console
+npm install --save font-awesome
+or
```
Comment (alexfedoseev) [inline docs/usage-webpack2.md]:
```shell
# or
```

to gray this out.

**Review by alex35mil — COMMENTED** [review]
I haven't had a chance to use this loader, but docs lgtm 👍  Just a few tiny comments.

**Comment by diondirza** [comment]
fixed with [2863733](https://github.com/shakacode/font-awesome-loader/pull/10/commits/286373359055b244b1beb3b0cf6719ea3f5fdbee)

**Comment by justin808** [comment]
@alexfedoseev Let me know if we should merge...I can try to check this out tomorrow as well.

**Comment by justin808** [comment]
@pksjce can you try this one out?

**Comment by alexfedoseev** [comment]
Sry, guys, I'll try to check this tomorrow or on Sunday.

**Comment by justin808** [comment]
@diondirza Can you please resolve the conflicts and I'll merge.

**Comment by diondirza** [comment]
got busy at this moment, later I will work on it.

**Comment by diondirza** [comment]
@justin808 done

**Comment by justin808** [comment]
:lgtm:

---

Reviewed 13 of 15 files at r3, 2 of 2 files at r4, 1 of 1 files at r6.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/shakacode/font-awesome-loader/10#-:-Kh58uodXYrXeYIMPPec:bnfp4nl)*

**Comment by justin808** [comment]
Thanks @diondirza!

**Pull request merged**